arweave/caip122.md

```diff
@@ -71,6 +71,8 @@ Type: ${type}
 Signature: ${signature}
 ```
 
+The signature creation process on Arweave is done through the `arweave-js` library. The data model is JSON serialized to a string. This string is signed using the RSA-PSS algorithm through a manual call to the `sign()` method from the `arweave-js` library. The signature is returned as a Uint8Array of bytes which is then Base64Url encoded by calling `arweave.utils.bufferTob64Url()`.
```

Comment:
message above will already be a string, would you then be able to directly sign that after or would you pass a Uint8Array of the string to be signed?

Reply:
If the data model is already a JSON serialised string then we can directly pass it to the sign() method, getting the Uint8Array which would be encoded to Base64 URL format to get the final signature which is the format that is needed to be sent over HTTP to the server for verification.

Reply:
@zachferland updated the PR to reflect the data model being a string.

Reply:
thanks! im good with the caip122 part 👍

Reply:
@oed if CAIP122 was the last part we needed to get sorted, can we get this merged so that we can move onto the next steps?